lib/service/service.go

```diff
@@ -147,6 +147,18 @@ import (
 	"github.com/gravitational/teleport/lib/web"
 )
 
+func init() {
+	backend.MustRegister(lite.New, lite.GetName())
```

Comment:
Shouldn't each backend register itself using a init function? It seems much simpler than running a init func in service

Reply:
Yeah for some reason I was thinking there was a circular dependency but that now I realize each backend implementation already imports the backend package. Maybe I was trying to init each backend in the backend package initially which gave the circular dependency. Anyways this makes sense I will fix it.

Reply:
Initializing each backend in its own package won't really work unless something (like `service.go`, or `e/tool/teleport/main.go`) imports the package, even just as a blank import (i.e. `import _ "github.com/gravitational/teleport/lib/backend/pgbk"`)

Reply:
I moved things around in [a5b4497](https://github.com/gravitational/teleport/pull/35973/commits/a5b44972eb3327909a8819a1ac94161ed57dbc71) and added blank imports where needed.